robot_smach_states/src/robot_smach_states/navigation/guidance.py

```diff
@@ -0,0 +1,210 @@
+"""
+Module contains states to guide an operator to a designated location.
+"""
+
+# ROS
+import rospy
+import smach
+import PyKDL as kdl
+
+# Robot skills
+from robot_skills.util.kdl_conversions import VectorStamped
+
+# robot_smach_states.navigation
+import navigation
+from robot_smach_states.navigation.navigate_to_symbolic import NavigateToSymbolic
+
+
+def _detect_operator_behind_robot(robot, distance=1.0, radius=0.5):
+    """
+    Checks if a person is within <radius> of the position <distance> behind the <robot>
+
+    :param robot: (Robot) api object
```

Comment:
You can also do 
```
"""
:type robot: robot_skills.Robot
:rtype: bool
"""
etc.

Reply:
Added typehints in 84116af41cb5a344bd640072fc81665ba6393be9